test/built-ins/Symbol/is-constructor.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-symbol-constructor
+description: >
+  Symbol is a constructor and has [[Construct]] internal method.
+info: |
+  The Symbol constructor:
+  [...]
+  * may be used as the value of an extends clause of a class definition
+  but a super call to it will cause an exception.
+  [...]
+  * has a [[Prototype]] internal slot whose value is %Function.prototype%.
+includes: [isConstructor.js]
+features: [Symbol, Reflect.construct]
+---*/
+
+assert(isConstructor(Symbol));
```

Comment:
JSC doesn't define `[[Construct]]` method for `Symbol`, yet `class extends Symbol` works because [it doesn't perform `IsConstructor`](https://bugs.webkit.org/show_bug.cgi?id=193057).